blockchain/init_derive_sha.go

```diff
@@ -17,23 +17,9 @@
 package blockchain
 
 import (
-	"github.com/klaytn/klaytn/blockchain/types"
-	"github.com/klaytn/klaytn/storage/statedb"
+	"github.com/klaytn/klaytn/blockchain/types/derivesha"
 )
 
 func InitDeriveSha(deriveShaImpl int) {
```

Comment:
Do we need this method? even this file?

Reply:
Yes, it's necessary and it must be here.

- `derivesha.InitDeriveSha` must be called from somewhere.
- `derivesha.InitDeriveSha` cannot be called from blockchain/types/derive_sha.go
because that would create an import cycle `types -> derivesha -> statedb -> types`.
- `derivesha.InitDeriveSha` could be called from elsewhere, but I decided to preserve the existing `blockchain.InitDeriveSha` for simplicity.

Reply:
Ah, thanks for the explanation.